### ISSUE TITLE
button padding in IE

### ISSUE THREAD

**Issue by jscharett** [US-CBP/cbp-theme#187]
The following style causes buttons to be askew in IE
```
.btn, .switch-toggle a, .ui-datepicker button.ui-state-default, ::-ms-backdrop {
    padding: 7px 12px 8px;    
    top: 1px;
}
```

![image](https://user-images.githubusercontent.com/7661580/34308948-951ba570-e71d-11e7-8d4e-a4e291adcd32.png)

**Comment by r1teshkumar** [comment]
@jscharett , what is the HTML markup that you have used here.

**Comment by jscharett** [comment]
I believe it followed the standard bootstrap markup using input-group and input-group-btn.  Will confirm and provide markup once I get a chance

**Comment by jscharett** [comment]
Here is the markup

```
<div class="input-group">
    <input type="text" class=" form-control">
    <div class="input-group-btn">
        <button class="btn btn-default" type="button">
            <span aria-label="Lookup"><i aria-hidden="true" class="fa fa-search"></i></span>
        </button>
    </div>
</div>
```

**Comment by r1teshkumar** [comment]
Thanks

**Comment by davezen1** [comment]
Resolved by PR #194
